src/uiStrings.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
I saw there is other constants defined in package.nls.json. Are these two solution work together or we need to use only one for later maintanance?

Reply:
According to this sample
https://github.com/microsoft/vscode-extension-samples/tree/main/l10n-sample
the package.nls.json works for package.json file, and the uistrings works for extension's src code

Reply:
reference API Center localization logic
https://github.com/microsoft/vscode-azureapicenter/pull/76
the package.nls.json is work for package.json, and this uistrings is work for ts files
there is another guide for localization for vs code ext
https://www.eliostruyf.com/localization-visual-studio-code-extensions/